UI/src/i18n.js

```diff
@@ -0,0 +1,91 @@
+/** @format */
+
+import { nextTick } from "vue";
+import { createI18n } from "vue-i18n";
+
+// It should be dynamic
+export const SUPPORT_LOCALES = [
+    "ar_EG",
+    "bg",
+    "ca",
+    "ckb",
+    "cs",
+    "da",
+    "de_CH",
+    "de",
+    "el",
+    "en_CA",
+    "en_GB",
+    "en",
+    "en_NZ",
+    "es_AR",
+    "es_CO",
+    "es_EC",
+    "es",
+    "es_MX",
+    "es_PA",
+    "es_PY",
+    "es_SV",
+    "es_VE",
+    "et",
+    "fa_IR",
+    "fi",
+    "fr_BE",
+    "fr_CA",
+    "fr",
+    "hu",
+    "id",
+    "is",
+    "it",
+    "lt",
+    "lv",
+    "ms_MY",
+    "nb",
+    "nl_BE",
+    "nl",
+    "pl",
+    "pt_BR",
+    "pt",
+    "ru",
+    "sv",
+    "tr",
+    "uk",
+    "zh_CN",
+    "zh_TW"
+];
+
+export function setI18nLanguage(i18n, locale) {
+    if (i18n.mode === "legacy") {
```

Comment:
As we only run on Vue3, does it make sense to support "legacy" mode?

Reply:
No, it doesn't. Fixed